calliostro/last-fm-client-bundle/2.0/manifest.json

```
@@ -3,10 +3,11 @@
         "Calliostro\\LastfmBundle\\CalliostroLastfmBundle": ["all"]
     },
     "copy-from-recipe": {
-        "config/": "%CONFIG_DIR%/"
+        "config/": "%CONFIG_DIR%/packages/"
```

```suggestion
        "config/": "%CONFIG_DIR%/"
```

As far as I understand, this change will put `config/packages/calliostro_lastfm.yaml` into `config/packages/packages/calliostro_lastfm.yaml`